README.md

```diff
@@ -1 +1,22 @@
 # hw01
+
+I am **Maria Jose Camacho**, second year student at Harris Public Policy School. Before coming to Chicago, I have worked in different agencies in order to define roles, functions and responsibilities at the regulatory, individual and institutional levels to support in the transition of the **criminal justice system** implemented in 20016 in Mexico. 
+
+
+# Work experience
+
+
+## Federal Attorney General’s Office
+
+
+* In my first job after graduation as **federal prosecutor**. I was responsible for defining and establishing the rules required to implement workflow models for the internal processes of each stage within a criminal case under this new legal system. I was also involved in creating criminal investigation protocols, management models for the agency’s **Internal Affairs Office**, and writing legislation that would later become the national guidelines for the new criminal procedure. In addition, I participated in reviewing evidence for *high-impact investigations* involving organized crime and kidnapping. We were tasked with challenging the existing roles and practices associated with the old inquisitorial system.
```

Comment:
I like how you included clear descriptions of your work experience and emphasized specific portions such as your job title. I think you might be missing some words in your first sentence and I would recommend rewording the first sentence a bit as the current phrasing is a bit confusing (e.g. "In my first job after graduation, I worked as a federal prosecutor.").